OBlock/contracts/StudentContract.sol

```diff
@@ -0,0 +1,21 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+contract StudentContact{
+
+    mapping(string => Student) studentMap;
+
+    struct Student{
+        uint256 id;
+        uint8 qca;

```

Comment:
just thinking here, we shouldn't have to add a students qca? that should be handled elsewhere (and potentially outside the scope for the project)

Reply:
Agree

Reply:
Fixed!!!!